tests/unit_test/channels/whatsapp_test.py

```
@@ -391,6 +391,103 @@ async def test_interactive_nfm_reply_with_documents_triggers_multimedia(self, mo
+        monkeypatch.setattr(
+            Whatsapp,
+            "get_business_phone_number_id",
+            lambda self: "142427035629239"
+        )
+
+        # Simulate save_whatsapp_media_content returning a list of one media id
+        s3_url = "https://uat-kairon-upload.s3.amazonaws.com/user_media/698431b7f85e2534c76f5034/919515991685_019c74a78760760fa2c08e4da2ce35c1_whataspp_360_885215267637065.jpeg"
+        monkeypatch.setattr(
+            UserMedia,
+            "save_whatsapp_media_and_get_url",
+            lambda bot, sender_id, whatsapp_media_id, description, config: [whatsapp_media_id, s3_url]
```

Ruff (0.15.1)

[warning] Unused lambda argument: `bot`

(ARG005)